paper-api/src/main/java/io/papermc/paper/world/biome/effects/AdditionSound.java

```diff
@@ -0,0 +1,24 @@
+package io.papermc.paper.world.biome.effects;
+
+import org.bukkit.Sound;
+import org.jetbrains.annotations.Range;
+
+/**
+ * Settings for additions sound.
```

Comment:
What are addition sounds.
Wiki link?
Explanation?

Reply:
Tbh can't be sure what it is to make a better explanation, the only 5 vanilla usages are
- {"sound":"minecraft:ambient.basalt_deltas.additions","tick_chance":0.0111}
- {"sound":"minecraft:ambient.crimson_forest.additions","tick_chance":0.0111}
- {"sound":"minecraft:ambient.nether_wastes.additions","tick_chance":0.0111}
- {"sound":"minecraft:ambient.soul_sand_valley.additions","tick_chance":0.0111}
- {"sound":"minecraft:ambient.warped_forest.additions","tick_chance":0.0111}

They really seems like an addition to other sounds, and wiki doesn't have any more info than what's in this javadoc